code/ship/ship.cpp

```diff
@@ -265,7 +265,8 @@ flag_def_list_new<Model::Subsystem_Flags> Subsystem_flags[] = {
 	{ "don't autorepair if disabled", Model::Subsystem_Flags::No_autorepair_if_disabled,        true, false },
 	{ "share fire direction",       Model::Subsystem_Flags::Share_fire_direction,               true, false },
 	{ "no damage spew",             Model::Subsystem_Flags::No_sparks,                          true, false },
-	{ "no impact debris",           Model::Subsystem_Flags::No_impact_debris,                    true, false },
+	{ "no impact debris",           Model::Subsystem_Flags::No_impact_debris,                   true, false },
+	{ "hide turret from ship summary", Model::Subsystem_Flags::Hide_turrets_in_loadout_stats,   true, false },
```

Comment:
There's a difference between the quoted name and the flag name.  Did you want to sync those up?

Reply:
Good point, just updated. Would be happy to continue to refine if needed.

Reply:
Looks good, although you missed one spot. ;)

Reply:
Opps, forgot to press save for each file before committing.